core/lib/include/xfa.h

```diff
@@ -108,7 +111,28 @@ _Pragma("GCC diagnostic ignored \"-Warray-bounds\"")
  */
 #define XFA_USE_CONST(type, name) \
     extern const type name []; \
-    extern const type name ## _end []
+    extern const type name ## _end []; \
+    XFA_ACCESSORS(type, name) \
+    extern const unsigned __xfa_dummy
+
+/**
+ * @brief define accessor functions
+ *
+ * This is a helper functions defining `<name>_len()` and `<name>_get(n)`
+ * accessor functions for a given XFA.
+ *
+ * @internal
```

Comment:
The `@internal` makes it clear, but maybe make it extra clear, that this macro is not required to be called by the user but is done automatically by `XFA_USE_*`. I was a bit confused at first, why `XFA_ACCESSORS()` did not show up in the unittests.

Reply:
Note that `@internal` only controls visibility and with our current doxygen settings it will be included in the generated HTML. The reader of the HTML will have no clue that this function is intended to be for internal use only. Maybe adding an explicit message via `@warning` can be done in addition?

Reply:
Welp, I read the doc also in code :-). I think what primarily confused me, that it is in the middle with the more public-fronting macros of `xfa`. At the risk of bike-shedding this: @kaspar030 can it maybe go more to the bottom? From how you use it, it doesn't seem like it's position is important.